templates/blog/index.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+
+{% block title %}
+  <title>Index</title>
+{% endblock %}
+
+{% block content %}
+  <h1>Hello, welcome to the blog!</h1>
+  <h2>All posts:</h2>
+  <br>
+  <ul>
+    {% for post in post_list %}
+      <li>
+        <a href="{% url 'blog:post-detail' post.id %}"><strong>{{ post.title }}</strong></a>
+        <br>
+        <em>by {{ post.owner.username }}</em> on {{ post.created_time|date:"F d, Y H:i" }}<br>
+        <p>{{ post.content }}</p>
+        <p>{{ post.commentary_count }}</p>
```

Comment:
It seems like you are trying to display the number of comments for each post with `{{ post.commentary_count }}`. Ensure that `commentary_count` is a valid attribute of the `post` object. If your model does not have this attribute, you might need to calculate it in your view and pass it to the template.